kit/src/test/java/com/oracle/javafx/scenebuilder/kit/skeleton/SkeletonBufferTest.java

```diff
@@ -54,7 +54,7 @@ public void testControllerWithoutPackageName() throws IOException {
         String skeleton = skeletonBuffer.toString();
 
         // then
-        String firstLine = skeleton.substring(0, skeleton.indexOf("\n"));
+        String firstLine = skeleton.substring(0, skeleton.indexOf(AbstractSkeletonCreator.NL));

```

Comment:
Better now. 
Since you have a single `NL` entry, have you tried setting `NL = "\n"` ? Does it pass the tests?

Reply:
When running with "\n", tests will pass on MacOS and Linux. On Window tests will fail.